Sources/Persistence/PersistencePerformanceMetricsTracker.swift

```diff
@@ -105,7 +105,7 @@ public extension PersistencePerformanceMetricsTracker {
             try execute { result in
 
                 switch result {
-                case .success(let blobSize, let memSize):
+                case .success((let blobSize, let memSize)):
```

Comment:
I think you should exclude this file's changes from this PR, as it's Xcode 11.4 (Swift 5.2) only 😉